apps/real-time-collaboration/server/AppInstanceLambda/index.js

```diff
@@ -0,0 +1,74 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+'use strict';
+const AWS = require('aws-sdk');
+var response = require('cfn-response');
+AWS.config.update({region: process.env.AWS_REGION});
+const chime = new AWS.Chime({region: process.env.AWS_REGION});
+
+exports.handler = async (event, context) => {
+  console.log('Event: \n', event);
+  console.log('Create Chime SDK App Instance');
+
+  try {
+    if (event["RequestType"] !== "Create") {
+      await response.send(event, context, response.SUCCESS, {});
+    }
+
+    const dateNow = new Date();
+    let chime_response;
+    //create a chime app instance
+    const params = {
+      Name: 'AWSChimeMessagingSDKDemo-' + dateNow.getHours().toString() + dateNow.getMinutes().toString(),
+    };
+    chime_response = await chime.createAppInstance(
+      params,
+      function (err, data) {
```

Comment:
You used two patterns-a promise and a callback-simultaneously. Recommend using the `.promise()` only. You can rewrite this to:

```
try {
  chimeResponse = await chime.createAppInstance(params).promise();
  console.log(chimeResponse);
} catch (error) {
  console.log(err, err.stack);
}
```

We can refactor this later.

Reply:
great. Thanks.